modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -7,9 +7,13 @@
 
 The OVN-Kubernetes Container Network Interface (CNI) cluster network provider has the following limitations:
 
-* OVN-Kubernetes does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`.
-The default value, `cluster`, is supported for both parameters.
-This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
+* OVN-Kubernetes does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`. The default value `cluster` is supported for both parameters. This limitation can affect you when you add a service of type LoadBalancer, NodePort, or add a service with an external IP.
+
+[NOTE]
+====
+Openshift Installer provisioned infrastructure(IPI) on Azure cloud, installer create Loadbalancer service with `ExternalTrafficPolicy` pointing to local, however the load is still distributed to the available backends until the fix is available, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1772847[BZ#1903408].
```

Comment:
It shouldn't matter whether it's IPI or UPI, and the issue would apply to other platforms as well as Azure.  

The link URL has one BZ number, and the link text has a different number.  I think you want the latter: <https://bugzilla.redhat.com/show_bug.cgi?id=1903408>.  

Is the intent here to add a BZ citation, or is there additional information that isn't covered by the existing note?